conans/test/client/cmd/export_test.py

```diff
@@ -0,0 +1,81 @@
+
```

Comment:
We are adding command tests to ``command`` test subfolder.

Reply:
Please, suggest a destination for this TestCase, `conans/test/command/unittest/export_test.py`?